api-lib/event_triggers/createReactionInteractionEvent.ts

```diff
@@ -27,6 +29,9 @@ export default async function handler(req: VercelRequest, res: VercelResponse) {
         created_at: created_at,
         reaction_id: reaction_id,
         reaction: reaction,
+        hostname: req.headers?.host || '',
+        big_question_id,
+        private_stream,
```

Comment:
i'd prefer these fields are explicitly defined `big_question_id: big_question_id` so they are clearly null or undefined rather than missing for events without them @sshmm